j2me/deprecated/org.javarosa.patient/src/org/javarosa/model/GraphDataGroup.java

```diff
@@ -20,7 +20,7 @@
 import java.util.Enumeration;
 import java.util.Vector;
 
-import org.javarosa.core.model.IDataReference;
+import org.javarosa.model.xform.XPathReference;
```

Comment:
We can likely just delete that directory now. Should add to the tech debt
roadmap, that code is dead dead.

On Thu, Apr 16, 2015 at 1:25 PM, Phillip Mates <notifications@github.com>
wrote:

> In
> j2me/deprecated/org.javarosa.patient/src/org/javarosa/model/GraphDataGroup.java
> <https://github.com/dimagi/javarosa/pull/73#discussion_r28530647>:
>
> > @@ -20,7 +20,7 @@
> >  import java.util.Enumeration;
> >  import java.util.Vector;
> >
> > -import org.javarosa.core.model.IDataReference;
> > +import org.javarosa.model.xform.XPathReference;
>
> Should I be concerned about modifying files in the 'deprecated' directory?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/javarosa/pull/73/files#r28530647>.
>